README.md

```diff
@@ -28,9 +28,9 @@ See the [documentation](docs/README.md) for more information.
 
 # Development
 
-Checkout the repository and get a development environment with `script/setup`. This will create a new virtual environment in the `venv` directory of the repository, and install all necessary requirements.
+Checkout the repository and get a development environment with `script/setup`. This will create a new virtual environment in the `venv` directory of the repository, and install all necessary requirements. It will also build your code space to install latest dependencies, so you should run it again occasionally.
```

Comment:
this is not correct. You either use a codespace, which does `script/setup` automatically.

We better remove this whole section and point at https://github.com/home-assistant/intents/blob/main/docs/codespace/README.md

Reply:
So one is not supposed to develop on their own machine, but on github codespaces instead? Interesting 🤔 

Is the local developing something that's frowned upon?

Reply:
@balloob even if you use codespaces, they only run `script/setup` upon provisioning. If you reuse them, you still need to run it manually when dependencies change.

I've personally suggested this a few times on Discord to people struggling with their setup and I think it's not a bad idea to have it in the README.

Reply:
Yes, running script/setup should be mentioned, because beginners will never re-run setup a thus will never update dependencies.
It might be good to know, that when some is starting new branch and there were some merges in the main brach, it is recommended to re-run setup to use latest updates to dependencies.

Reply:
People need to just rebuild their code space to get the latest dependencies. That's what it is for. They should not have to learn development commands about setting up environments.

Reply:
You are totally fine to run this on your own machine. However, 99% of the people contributing to this repository is not a developer and doesn't know any of this. Adding instructions will only scare them away. Instead, we should push codespaces as that's doing everything for people out of the box.

Reply:
Then can we at least mention that you need to rebuild your codespace for each contribution?

Reply:
For a code space we could add a task to install requirements and mark it as [`runOn: folderOpen`](https://code.visualstudio.com/Docs/editor/tasks#_run-behavior) to keep it up to date.